### DISCUSSION TITLE
Same Message ID should stop extracting

### DISCUSSION THREAD

**Discussion by ladytrekker** [lingui/js-lingui#1235] [Q&A]
I need to stop the extract command from continuing updating the files when the same message ID was found.

So far it just logs "Cannot process file .... Different defaults for the same message ID." and continues.

The result are updated PO files and the doubled message IDs and strings are deleted from all files.

Is there any way to stop extracting in this case? Thanks.

**Comment by timofei-iatsenko** [comment]
Would be as you explaining after merging this https://github.com/lingui/js-lingui/pull/1358
